src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs

```diff
@@ -106,9 +106,9 @@ public void ListOfNonNull_containing_null_should_bubble_up_the_null()
             var data = new Data {ListOfStrings = new List<string> {"text", null, null}};
             var errors = new[]
             {
-                new ExecutionError("Cannot return null for non-null type. Field: listOfNonNullable, Type: [String!].")
+                new ExecutionError("Error trying to resolve listOfNonNullable.")
                 {
-                    Path = new object[] {"nonNullableDataGraph", "listOfNonNullable", 1}
+                    Path = new object[] {"nonNullableDataGraph", "listOfNonNullable"}
```

Comment:
The list index is lost from the Path property in this particular case.  However, that matches the updated message, as in this case the list is a list of non-nullable objects, and the entire object was unable to be resolved.  I have added the index to the message property of the inner exception.  See test `list_throws_for_invalid_type_when_conversion_returns_null`.  Note that if the list was nullable, then the error should be returned for the particular item in the list, and that item set to null.  But this is not yet supported by the graphql infrastructure, as noted in the PR.

Reply:
> Note that if the list was nullable

Do you mean _if the list **elements** were nullable_ ?

Reply:
OK